### ISSUE TITLE
loss becomes nan in tutorial ‘fit textured mesh’

### ISSUE THREAD

**Issue by 315930399** [facebookresearch/pytorch3d#964]
when I ran _3.Mesh and texture prediction via textured rendering_ in tutorial _fit textured mesh_, the loss becomes nan in the early stage of training.

**Comment by bottler** [comment]
This has been reported a few times and the master issue is #561. It should be fixed in the latest code in the main branch which you could use instead of a release either by building from source or using a nightly build. You can also add `perspective_correct=False` to the `RasterizationSettings` objects to avoid the problem.

**Comment by simon-cross** [comment]
I have installed PyTorch3D with the following command
`pip3 install "git+https://github.com/facebookresearch/pytorch3d.git"`
but, I still get nan when running the tutorial fit_textured_mesh.py.

Am I not using the code of the main branch?

**Comment by hajiejue** [comment]
I fix this problem by setting the perspective_correct=False
